.buildkite/scripts/steps/run-agentless-tests.sh

```diff
@@ -24,13 +24,17 @@ extract_sha() {
     local env=$1
     local sha
 
-    # Ensure repo is available
-    git clone --depth 1 git@github.com:elastic/serverless-gitops.git || (
-        cd serverless-gitops
-        git pull
-    )
 
-    go install github.com/mikefarah/yq/v4@v4.45.1
+
+    # Ensure repo is available - redirect output to /dev/null
+    if [ ! -d "serverless-gitops" ]; then
+        git clone --depth 1 git@github.com:elastic/serverless-gitops.git > /dev/null 2>&1
+    else
+        (cd serverless-gitops && git pull > /dev/null 2>&1)
+    fi
+
+
+    go install github.com/mikefarah/yq/v4@v4.45.1 > /dev/null 2>&1
```

Comment:
This will hide errors, making it hard to debug if the install fails. How about:

```suggestion
    go install github.com/mikefarah/yq/v4@v4.45.1 1>&2
```